src/software/backend/radio/robot_status.h

```diff
@@ -88,11 +88,11 @@ struct RobotStatus
 };
 
 /**
- * Converts RobotStatus to RobotStatusMsg
+ * Converts RobotStatus to TbotsProto::RobotStatus
  * Does not convert ChipperKickerStatus,
  * DriveUnits, and NetworkStatus due to insufficient info
  *
  * @param robot_status The RobotStatus
```

Comment:
Checking if `The RobotStatus` should be `The TbotsProto::RobotStatus` to match above.

Reply:
no, `RobotStatus` is a legacy struct for radio, while `TbotsProto::RobotStatus` is a proto msg

Reply:
renamed to RadioRobotStatus for clarity